src/components/workflows/WorkflowsControl.vue

```diff
@@ -285,7 +285,13 @@ async function runTask() {
     params.append(`property(${key})`, data.value[key])
   })
 
-  await fetch(url).then((response) => {
+  await fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded',
```

Comment:
Should we use `application/json`?

Reply:
https://publicwiki.deltares.nl/display/FEWSDOC/FEWS+PI+REST+Web+Service#FEWSPIRESTWebService-POSTruntask(2017.02), without application/x-www-form-urlencoded it does not work